<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpanel</class>
 <widget class="QDialog" name="adminpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/SADS.png</normaloff>:/resources/SADS.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_disabled">
      <property name="text">
       <string>Disabled
Seating</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_prices">
      <property name="text">
       <string>Change
Prices</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_maxSeats">
      <property name="text">
       <string>Maximum
Seats</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>adminpanel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
